user_guide/intro_company_structure_org_selector.rst

```diff
@@ -0,0 +1,198 @@
+access to the company’s French division.
+
+However, users can be assigned to multiple organizations just as they can be assigned to multiple business units and 
+will get access to the records in different organizations. For example, if you are working at the worldwide help desk 
+of a multinational organization, you should be able to access customer records in all the organization within your 
+company in order to open cases based on the customer requests. Similarly, if you are the CEO of a big enterprise 
+comprised of multiple independent divisions, you would like to have access to every one of them in order to check their 
+performance.
+
+As organizations are often much bigger and more separated than business units, the user will have to switch to the 
+necessary organization to work with data in each one of them.
+
+
+Switching Between Organizations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The name of an organization you are currently working with is displayed in the top left corner of the screen. If you 
+have access to multiple organizations, upon your first login you will see a note with your current organization. Three 
+dots to the left are the organization selector.
+
+|
+
+.. image:: ./img/multi_org/multi_org_select.png
+
+|
+
+Click the selector, to see a list of organizations you have access to. Your current organization will be displayed in 
+bold. If a System organization exists, it will be shown at the top of the list and shifted left
+
+|
+
+.. image:: ./img/multi_org/multi_org_choice.png
+
+|
+
+To switch to another organization, click its name in the list.
+
+
+Work with a System Organization
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A System Organization provides users with simultaneous access to all the records of all the organizations they are 
+assigned to. There cannot be more than one system organization in OroCRM. 
+
+When working in a system organization all grids will have an additional ‘Organization"* column. The *"Organization"* field 
+will be shown for each record viewed. The organization field is not editable.
+
+When creating a record in a system organization, the user should select an organization to which the record will belong.
+
+|
+
+.. image:: ./img/multi_org/multi_org_system1.png
+
+|
+
+.. note::
+
+    A record can belong to the system organization as well — if this is the case, the record won’t be accessible from 
+    anywhere else other than the system organization itself. One example of such a record may be an aggregate report 
+    based on data from multiple organizations — it will make sense only within the system organization where all the 
+    necessary data is available.
+
+Conclusion
+----------
+
+Organizations and business units to which users belong will define the data available to them, and will mirror the 
+company structure and related rules from the physical world to the software application.
```

Comment:
Edit: Organizations and business units to which users belong will define the data available to them, and will mirror the real-world company structure and related rules.